src/Cinelovers.Core/Mappers/MovieMapper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Cinelovers.Core.Rest.Dtos;
+using Cinelovers.Core.Services.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace Cinelovers.Core.Mappers
+{
+    public sealed class MovieMapper
+    {
+        const string PosterBaseUrl = "https://image.tmdb.org/t/p/";
```

Comment:
This one and the main URL would probably be better in a single, separate, `ApiConstants` file, no?

Reply:
It could be moved but since it's being used in a single place and everytime I map a movie from the API to the domain one it will depend on this mapper I can't see much troubles with that.